community/kernel/src/test/java/org/neo4j/kernel/impl/api/index/IndexPopulationJobTest.java

```diff
@@ -640,13 +644,14 @@ private IndexPopulationJob newIndexPopulationJob( FailedIndexProxyFactory failur
         return job;
     }
 
-    private IndexDescriptor indexDescriptor( Label label, String propertyKey, boolean constraint ) throws TransactionFailureException
+    private IndexDescriptor indexDescriptor( Label label, String propertyKey, boolean constraint )
+            throws TransactionFailureException, IllegalTokenNameException, TooManyLabelsException
     {
-        try ( KernelTransaction tx = kernel.newTransaction( KernelTransaction.Type.implicit, AnonymousContext.read() );
+        try ( KernelTransaction tx = kernel.newTransaction( KernelTransaction.Type.implicit, AnonymousContext.AUTH_DISABLED );
```

Comment:
Since now we do not allow creation of index descriptors for non-existent properties and labels we should be able to create them if they are not explicitly created by tests and to be able to do that we can't use `AnonymousContext.read()`. And since it does not really matter for this test `AUTH_DISABLED` use instead here now